downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -893,14 +893,14 @@ bal openapi -i <ballerina file> --json
 
 #### Bindgen Tool
 
-- Improve the generated bindings with the use of distinct type classes.
-- Improve the internal mechanism used to generate the bindings. Previous handlebars-based implementation is now changed to a syntax-tree-based implementation.
+- Improved the generated bindings with the use of distinct type classes.
+- Improved the internal mechanism used to generate the bindings. Previous handlebars based implementation is now changed to a syntax tree based implementation.
```

Comment:
It's better to keep these words hyphenated:  syntax-tree-based and handlebars-based

Reply:
Fixed